central/compliance/data/repository.go

```diff
@@ -203,6 +203,7 @@ func policyCategories(policies []*storage.Policy) map[string]set.StringSet {
 }
 
 func (r *repository) init(ctx context.Context, domain framework.ComplianceDomain, scrapeResults map[string]*compliance.ComplianceReturn, f *factory) error {
+	log.Infof("Init'ing repo")
```

Comment:
nit:  Did you mean to leave this here?  (I'm fine leaving it, just making sure you meant to)

Reply:
Nope, thanks